sdk/storage/azdatalake/filesystem/constants.go

```diff
@@ -0,0 +1,17 @@
+//go:build go1.18
+// +build go1.18
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+package filesystem
+
+import "github.com/Azure/azure-sdk-for-go/sdk/storage/azblob"
+
+// PublicAccessType defines values for AccessType - private (default) or file or filesystem.
+type PublicAccessType = azblob.PublicAccessType
+
+const (
+	File       PublicAccessType = azblob.PublicAccessTypeBlob
+	Filesystem PublicAccessType = azblob.PublicAccessTypeContainer
```

Comment:
```suggestion
	FileSystem PublicAccessType = azblob.PublicAccessTypeContainer
```

Reply:
Should be Filesystem https://learn.microsoft.com/en-us/rest/api/storageservices/datalakestoragegen2/filesystem